CHANGELOG.md

```diff
@@ -111,30 +111,30 @@ Return argument value as field definition runtimeType if requested as object typ
 ### Added
 
 - Additional HotChocolate scalars  (HotChocolate.Types.Scalars)
-    - PhoneNumber (#2995)
```

Comment:
ℹ️ My auto-save formatting appears to have made some additional minor changes here. I don't think it's an issue but if you need me to revert these specific changes, I will.